license.txt

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2021 IntelliTect
```

Comment:
I don't know how much it may matter legally, but the company's name (at least according to my W-2) is still "IntelliTechture"

Reply:
![image](https://user-images.githubusercontent.com/105415/113789851-d7b3bf00-96f4-11eb-932b-8e1abfb2cdda.png)

Reply:
I'd be very surprised if we didn't have a DBA for the IntelliTect name, which I'd think would be legally sufficient in this case.